openshift/osd-network-verifier
# osd-10310: Creating AWS infrastructre templates for osd-network-verifier testing

PR content:
a go script to create a VPC and firewall for OSD-network-verifier testing.
How to test:
run aws sts assume-role --role-arn arn:aws:iam::"$your-aws-account-number":role/OrganizationAccountAccessRole --role-session-name "$your-name" --profile osd-staging-2
run go builld awsconfig.go
AWS_ACCESS_KEY_ID= AWS_SECRET_ACCESS_KEY= AWS_SESSION_TOKEN= ./firewallConfig
open your AWS console and see that VPC and Firewall have been created.
